Tests/Microsoft.AppCenter.Crashes.Test.Windows/CrashesTest.cs

```diff
@@ -69,6 +70,33 @@ public void InstanceIsNotNull()
             Assert.IsNotNull(Crashes.Instance);
         }
 
+        [TestMethod]
+        public void VerifyRemoveMemoryWarningAfterStart()
+        {
+            var mockMemoryWarningHelper = new Mock<IMemoryWarningHelper>();
+            AppCenter.Instance.ApplicationSettings.SetValue(Crashes.PrefKeyMemoryWarning, true);
+            Crashes.Instance = new Crashes();
+            Crashes.Instance._memoryWarningHelper = mockMemoryWarningHelper.Object;
+            Crashes.Instance.OnChannelGroupReady(_mockChannelGroup.Object, string.Empty);
+            Crashes.SetEnabledAsync(true).Wait();
+            _mockApplicationLifecycleHelper.Raise(eventExpression => eventExpression.UnhandledExceptionOccurred += null,
+                new UnhandledExceptionOccurredEventArgs(new System.Exception("test")));
+            var hasMemoryWarning = Crashes.HasReceivedMemoryWarningInLastSessionAsync().Result;
+            Assert.IsTrue(hasMemoryWarning);
+            Assert.IsFalse(AppCenter.Instance.ApplicationSettings.GetValue(Crashes.PrefKeyMemoryWarning, false));
+        }
+
+        [TestMethod]
+        public void VerifyRemoveMemoryWarningAfterDisable()
+        {
+            Crashes.Instance = new Crashes();
+            Crashes.Instance.OnChannelGroupReady(_mockChannelGroup.Object, string.Empty);
+            Crashes.SetEnabledAsync(true).Wait();
+            AppCenter.Instance.ApplicationSettings.SetValue(Crashes.PrefKeyMemoryWarning, true);
+            Crashes.SetEnabledAsync(false).Wait();
+            Assert.IsFalse(AppCenter.Instance.ApplicationSettings.GetValue(Crashes.PrefKeyMemoryWarning, false));
```

Comment:
Can you also add one more check here that the key is removed?
`AppCenter.Instance.ApplicationSettings.ContainsKey(Crashes. PrefKeyMemoryWarning)` should return `false`.